Foundation/NSError.swift

```diff
@@ -10,6 +10,14 @@
 //
 //===----------------------------------------------------------------------===//
 
+#if canImport(MSVCRT)
+import MSVCRT
+import WinSDK
```

Comment:
Do you really need the Windows SDK?  What are you using from there?  I think that you can get away with `import ucrt`.

Reply:
I honestly had no idea how to use the WinSDK, so I just used what I saw you put in other places. What is Window's equivalent of POSIX Error codes? @compnerd

Reply:
Ah, POSIX Error Codes?  That is from the Windows SDK.  It is the Windows Error Codes.  https://docs.microsoft.com/en-us/windows/desktop/debug/system-error-codes